### PR TITLE
Switch to TinyMCE

### PR DESCRIPTION
**Pull request by mgwalker**
Opening this PR so folks can see how the app looks/works with TinyMCE and we can start documenting any dealbreakers.

### This pull request changes...

- expected change 1
- goes with #1871

### This pull request is ready to merge when...

- [x] Tests have been updated (and all tests are passing)
- [x] This code has been reviewed by someone other than the original author
- [x] The experience passes a basic manual accessibility audit (keyboard nav, screenreader, text scaling) OR an exemption is documented
- [x] Changelog is updated as appropriate

### This feature is done when...

- [ ] Design has approved the experience
- [ ] Product has approved the experience

### CODE REVIEW

**Comment by jeromeleecms** [comment]
Well this is neat... tables and bullets from Word copied in well.

image

Images copied in as well (though it doesn't seem like there is an option in the toolbar to add it - you just copy the image in).. and yes, this is an image of the tables and bullets I just copied in...

image

**Comment by mgwalker** [comment]
noice

If we want a button for uploading images, we'll have to do that ourselves, but it looks like it should be pretty straightforward. We might need to disable drag-and-drop/copy-paste images in order to support separate uploads, but I'll need to spend some more time looking at the paste plugin. It _might_ let us handle image pastes magically, in which case 🙌.

I've also noticed that spellcheck is a little funky. It highlights misspelled words, but when you right-click them, it offers to turn them into links. So that's not super helpful. I'm not sure how to fix that, but I also haven't looked into it yet so maybe it's super simple.

**Comment by mgwalker** [comment]
Fixed the spellcheck thing by just taking out the "links" plugin - I don't think it makes sense _for now_, since we're generating PDFs for submission. PDFs can have links, but they're pointless once you print it out. 😛

Also figured out that we can add an `images_upload_handler` prop to the Editor to handle uploading images and replacing them with URLs. Pretty straightforward, so that should play well with #1735.

**Comment by mgwalker** [comment]
### Issue

 When you load the APD page, it can take a few extra seconds for the rich text editors to appear. TinyMCE loads each text box into an iframe by default, and since iframes cannot share styles, each instance must load its styles independently. The result is that we have dozens of requests for the same 3 CSS files, and none of the text boxes becomes visible until all of those requests are fulfilled.

We can make the boxes load much faster by putting them in inline mode. However, in this mode, the boxes become more like editable content than input elements – there is nothing to indicate that they can be edited at all until you just happen to click on one. We also cannot control the size of the box in inline mode – it will be as tall as the content, making the scrolling problem worse.

File: web/src/components/RichText.js [hunk 1]
```diff
@@ -1,69 +1,94 @@
...
-import { EDITOR_CONFIG, htmlToEditor, editorToHtml } from '../util/editor';
+// A theme is also required
+import 'tinymce/themes/silver';
+
+// Any plugins you want to use has to be imported
+import 'tinymce/plugins/paste';
+import 'tinymce/plugins/spellchecker';
+
+require.context(
+  'file-loader?name=[path][name].[ext]&context=node_modules/tinymce!tinymce/skins',
```
[the remark this answers is not in the document]
Comment (mgwalker) [reply web/src/components/RichText.js]: Yeah I don't know either.

**Pull request merged**